packages/picocad.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Picocad < Package                 # The first character of the class name must be upper case
+  description 'picoCAD is a program to build and texture lowpoly 3D models.'
+  homepage 'https://johanpeitz.itch.io/picocad'
+  version '1.0.1'
+  compatibility 'all'
+  license 'Proprietary'
```

Comment:
@saltedcoffii: What are your thoughts about this license?

Reply:
It needs to specify the TOS or EULA specifically. Just stating that a license is proprietary isn't enough.